python/tests/test_pyhdk_api.py

```diff
@@ -130,6 +130,36 @@ def test_import_csv(self, header, create_table, full_schema, glob):
 
         hdk.drop_table(table_name)
 
+    @pytest.mark.parametrize("create_table", [True, False])
+    @pytest.mark.parametrize("glob", [True, False])
+    def test_import_parquet(self, create_table, glob):
+        hdk = pyhdk.init()
+        table_name = "table1"
+
+        if create_table:
+            hdk.create_table(table_name, (("a", "int64"), ("b", "fp64")))
+            real_schema = {"a": "INT64", "b": "FP64"}
+
+        if glob:
+            file_name = ["omniscidb/Tests/ArrowStorageDataFiles/int_float.parquet", "omniscidb/Tests/ArrowStorageDataFiles/int_float.parquet"]
```

Comment:
This doesn't actually a glob test. You should use some file pattern to test it.

Reply:
changed test. We should clarify supported types and conversion mapping in further work.